clrenv/lazy_env.py

```diff
@@ -6,14 +6,15 @@
 from itertools import chain, groupby
 import shlex
 import sys
-import types
 
 from munch import Munch, munchify
 import yaml
 
 from .path import find_environment_path, find_user_environment_paths
 from functools import reduce
 
+import boto3
+SSM_CLIENT = boto3.client('ssm')
```

Comment:
should probably lazy-load this

Reply:
yeah definitely - you could memoize this inside a function to do that

Reply:
done, thanks